.env.test

```diff
@@ -10,3 +10,7 @@ SECRET_KEY=test_secret_key
 TRELLO_API_KEY=this_is_trello_api_key_123
 TRELLO_TOKEN=and_this_is_the_trello_token_56789
 BOARD_ID=5fb51906cc7d395b1b145b0e
+
+# Set Type of Storage To Use
+STORAGE_TYPE=TRELLO
+#STORAGE_TYPE=MONGO
```

Comment:
Ah, we don't really want to support both simultaneously. While I like your abstraction of the storage solution, in this case it would mean more work to maintain for no benefit.

The idea of the exercise was to rip out all of the Trello functionality and **replace** it with storage in a Mongo DB 

Setting STORAGE_TYPE=TRELLO in your .env.test means that your Mongo-related code is untested by integration tests.

Reply:
Good point. Didn't want to spend time adding integration tests for Mogo persistence as it didn't seem to part of the exercise. Will add later.